sklearn/tests/test_multiclass.py

```diff
@@ -155,13 +155,6 @@ def test_ovr_ovo_regressor():
     # we are doing something sensible
     assert np.mean(pred == iris.target) > 0.9
 
-    ovr = OneVsOneClassifier(DecisionTreeRegressor())
-    pred = ovr.fit(iris.data, iris.target).predict(iris.data)
-    assert len(ovr.estimators_) == n_classes * (n_classes - 1) / 2
-    assert_array_equal(np.unique(pred), [0, 1, 2])
-    # we are doing something sensible
-    assert np.mean(pred == iris.target) > 0.9
-
```

Comment:
I think it should be fine to leave this in, since it tests not just simple input-validation but rather that the `ovr` estimator has the correct properties after predict (number of estimators) and that the prediction itself is valid.